python/lsst/jointcal/utils.py

```diff
@@ -17,13 +17,16 @@
 
 import lsst.log
 import lsst.afw.table
+from lsst.afw.image import fluxFromABMag, abMagFromFlux
 from lsst.afw.geom import arcseconds
 
+MatchDict = collections.namedtuple('MatchDict', ['relative', 'absolute'])
+
 
 class JointcalStatistics(object):
     """Compute statistics on jointcal-processed data, and optionally generate plots."""
```

Comment:
The calculations take quite a long time, so doing them at construction time is a problem. It wasn't really meant to be used as a way for a user to grab random statistics, but rather was lifted out of the testing code so that I can calculate the statistics and make plots independent of the unittests. As I say in the top-level comment, a lot of this should go away once we have a full-focal visit composite data structure and a better n-way matcher.

I have some of the statistics calculated as part of `make_plots()`, which I supposed I could move into a different method: the idea was that unittests have a very specific set of things they need calculated, but don't care about anything else (which can take longer), so limit the math in `compute_rms()` to what the unittests need and no more.

That said, I've added a comment about usage to the class docstring.